readme.md

```diff
@@ -1,56 +1,64 @@
+   a form field can only be mapped to one Paystation transaction field at a time.
+
+## Using Gravity Forms Paystation:
 
 Before you can use the plugin to make payments, you need to create an account with Paystation.
 To do this please see visit the [Paystation](https://www2.paystation.co.nz/) website.
 
 Then you will need to enter some information in the plugin settings
 
 1. Select 'Plugins' from the WordPress Admin menu.
-2. Click the 'Settings' link for the Gravity Forms Paystation (3 party hosted) plugin.
-3. Enter your Paystation Id in the Paystation Id field.
-4. Enter your Paystation Gateway Id in to the Paystation Gateway Id field.
-5. Enter a string of letters and numbers for the Security Hash (8 to 20 characters is fine).
-6. Save the settings by clicking the 'Save Settings' button.
-7. Email info@paystation.co.nz letting us know the following: that you are using the Paystation Gravity forms plugin, the url to your wordpress site, and what your Security Hash is.
-
-### Building a Gravity Form with Credit Card Payments
-
-1. Create a new Gravity form
-1. Add one or more Product fields or a Total field to your form. The plugin will automatically detect the values assigned to these pricing fields
-3. Return to Paystation (3 party) menu where you setup your Paystation settings above and click 'Setup Paystation Feed'
-4. Click 'Add New' feed at the top of the page
-5. Select your form from the dropdown at the top of the page and save
-6. If you want custom field mappings to Paystation data, you can map your form fields to Paystation transaction fields (Merchant Reference, Customer details, Order details), be aware
-   a form field can only be mapped to one Paystation transaction field at a time.
+2. Select the Gravity Forms plugin, the 3 party setting will be under the gravity forms plugin.
+3. And then choose the Paystation (3 party hosted).
+4. Enter your Paystation Id in the Paystation Id field, Gateway ID in the gateway Id field. 
+5. Enter a string of letters and numbers for the Security Hash (8 to 20 characters is fine). 
+6. Then choose 'No' in the test mode field. 
+7. Save the settings by clicking the 'Save Settings' button. 
+8. And then under the same setting click on setup Paystation feed, it will redirect to the feeds page.
+9. On the feeds settings page, select thye form name from the dropdown which yoy have created 
+10. Email [info@paystation.co.nz](mailto:info@paystation.co.nz) letting us know the following: that you are using the Paystation Gravity forms plugin, the url to your wordpress site, and what your Security Hash is.
+
+## Testing dev environment:
+
+Tested successfully with the latest WordPress 6.4 version, PHP 8.2 version, Gravity forms 2.8 version
+Please note: This plugion is compatbile with the latest version as mentioned above, any downgrade versions will occure an issue and this might break your website.
```

Comment:
```suggestion
Please note: This plugin is compatible with the latest version as mentioned above, any downgrade versions may introduce an issue and this might break your website.
```